### PR TITLE
Initial doc update before 0.8

### PR DESCRIPTION
**Pull request by ddworak**

### CODE REVIEW

File: guide/selenium/src/test/scala/io/udash/web/guide/demos/ext/I18nDemosTest.scala [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
-      val output = driver.findElementById("output")
-      additionalAction(output)
-      val translations = output.getText.split("\n").collect {
-        case line if line.indexOf(":") != -1 => (line.take(line.indexOf(":")), line.drop(line.indexOf(":") + 2))
-      }.toMap
-      translations.size should be(6)
-      translations
-    }
-  }
+  //private def runDemo(id: String)(additionalAction: WebElement => Unit = _ => ())(implicit position: Position): Map[String, String] = {
```
Comment (FreexD) [inline guide/selenium/src/test/scala/io/udash/web/guide/demos/ext/I18nDemosTest.scala]: do we really need it? How about some comment with todo then?

File: guide/guide/src/main/scala/io/udash/web/guide/StatesToViewFactoryDef.scala [hunk 2]
```diff
@@ -42,7 +42,7 @@ class StatesToViewFactoryDef extends ViewFactoryRegistry[RoutingState] {
 
       case RestState => MarkdownPageViewFactory[RestState.type]()(Context.serverRpc.pages)
 
-      case I18NExtState => MarkdownPageViewFactory[I18NExtState.type]()(Context.serverRpc.pages)
+      case I18NExtState => I18NExtViewFactory
```
Comment (FreexD) [inline guide/guide/src/main/scala/io/udash/web/guide/StatesToViewFactoryDef.scala]: I understand this is only temporary, until we figure out the `I18n` fiddles?

File: guide/selenium/src/test/scala/io/udash/web/guide/demos/ext/I18nDemosTest.scala [hunk 3]
```diff
@@ -47,26 +46,72 @@ class I18nDemosTest extends SeleniumTest {
...
-      val translations = runDemo("rpc-translations-demo")()
-      verifyPlTranslations(translations)
+
+      //val translations = runDemo("rpc-translations-demo")()
+      //verifyPlTranslations(translations)
+      def demo = driver.findElementById("rpc-translations-demo")
+
+      eventually {
+        val elements = demo.findElements(new ByCssSelector("li"))
+        val translations: Map[String, String] = elements.asScala.map(item => {
```
Comment (FreexD) [inline guide/selenium/src/test/scala/io/udash/web/guide/demos/ext/I18nDemosTest.scala]: this translations can be extracted to a method - it's used in 3 places

File: guide/project/Dependencies.scala [hunk 4]
```diff
@@ -62,8 +62,7 @@ object Dependencies {
   val guideJsDeps = Def.setting(Seq.empty[org.scalajs.sbtplugin.JSModuleID])
 
   val homepageJsDeps = Def.setting(Seq[org.scalajs.sbtplugin.JSModuleID](
-    ProvidedJS / "jquery.mCustomScrollbar.concat.min.js",
```
Comment (FreexD) [inline guide/project/Dependencies.scala]: what was this used for before?

File: guide/guide/src/main/scala/io/udash/web/guide/views/bootstrapping/BootstrappingIntroView.scala [hunk 5]
```diff
@@ -5,44 +5,42 @@ import io.udash.css.CssView
...
 
 class BootstrappingIntroView extends FinalView with CssView {
   import Context._
-
   import JsDom.all._
 
   override def getTemplate: Modifier = div(
     h2("Project structure"),
-    p("The basic Udash project contains three modules: "),
+    p("A typical Udash project contains three modules: "),
```
Comment (FreexD) [inline guide/guide/src/main/scala/io/udash/web/guide/views/bootstrapping/BootstrappingIntroView.scala]: `consist of`

File: guide/guide/src/main/scala/io/udash/web/guide/StatesToViewFactoryDef.scala (hunk 2 above)
Comment (ddworak) [reply guide/guide/src/main/scala/io/udash/web/guide/StatesToViewFactoryDef.scala]: We cannot use them until new RPC is in.

File: guide/selenium/src/test/scala/io/udash/web/guide/demos/ext/I18nDemosTest.scala (hunk 1 above)
Comment (ddworak) [reply guide/selenium/src/test/scala/io/udash/web/guide/demos/ext/I18nDemosTest.scala]: I wanted to save time for when new rpc is merged. Adding comment.

File: guide/selenium/src/test/scala/io/udash/web/guide/demos/ext/I18nDemosTest.scala (hunk 3 above)
Comment (ddworak) [reply guide/selenium/src/test/scala/io/udash/web/guide/demos/ext/I18nDemosTest.scala]: IDC as it's going to be removed anyway. The new version of this test extracts it.

File: guide/guide/src/main/scala/io/udash/web/guide/views/bootstrapping/BootstrappingIntroView.scala (hunk 5 above)
Comment (ddworak) [reply guide/guide/src/main/scala/io/udash/web/guide/views/bootstrapping/BootstrappingIntroView.scala]: `consists` but nice catch anyway

**Pull request merged**